src/main/resources/view/MainWindow.fxml

```diff
@@ -46,14 +47,45 @@
           </padding>
         </StackPane>
 
-        <VBox fx:id="personList" styleClass="pane-with-border" minWidth="340" prefWidth="340" VBox.vgrow="ALWAYS">
-          <padding>
-            <Insets top="10" right="10" bottom="10" left="10" />
-          </padding>
-          <StackPane fx:id="personListPanelPlaceholder" VBox.vgrow="ALWAYS"/>
-        </VBox>
+        <BorderPane VBox.vgrow="ALWAYS">
+          <left>
+            <BorderPane styleClass="pane-with-border" minWidth="200" maxWidth="200">
+              <top>
+                <Label text="Blocks &amp; Rooms" styleClass="label-reference-header">
+                  <BorderPane.margin>
+                    <Insets top="10" bottom="5" left="5" />
+                  </BorderPane.margin>
+                </Label>
+              </top>
+
+              <center>
+                <VBox styleClass="reference-list">
+                  <BorderPane.margin>
+                    <Insets left="5" />
+                  </BorderPane.margin>
+
+                  <Label text="Block A"/>
+                  <Label text="Rooms: 100 to 120" />
+                  <Label text="Block B"/>
+                  <Label text="Rooms: 100 to 120" />
+                  <Label text="Block C"/>
+                  <Label text="Rooms: 100 to 120" />
```

Comment:
Room follows this format
<Level><Room No>

Shouldnt the range from room include other levels other than 1?
eg. 100 - 420 (The block has 4 levels, 20 rooms per level)

Reply:
Okay! I updated it. Currently it's just dummy data. We should probably have a source of truth somewhere in the code to validate if the block numbers and room numbers are all valid (I'm thinking somewhere in `preferences.json` might be good).